doc/source/conf.py

```diff
@@ -68,7 +68,11 @@
     "contributors",  # custom pandas extension
 ]
 
-exclude_patterns = ["**.ipynb_checkpoints"]
+exclude_patterns = [
+    "**.ipynb_checkpoints",
+    # https://github.com/sphinx-doc/sphinx/issues/1965#issuecomment-124732907
```

Comment:
Can you in addition to the link also put a short explainer what this is about?

Reply:
Done.